### PR TITLE
Added an 'easy-to-use' Show/Hide Framework: util.js

### PR DESCRIPTION
**Pull request by npetryk**
I've made util.js and used it in news.html to show off a show/hide framework I whipped together. I think we can cut back on a lot of nasty javascript if we can abstract some of the things we're going to want to do often. 

Theres more documentation in util.js, but it basically allows you to easily define 'hideable' elements and  'trigger' elements that hide or show the 'hideable' element when clicked on. It's not extremely feature rich but it provides a nice little framework with decent customization capacity that I could see us using all over the place.

### CODE REVIEW

File: app/views/Application/news.html [hunk 1]
```diff
@@ -36,30 +36,46 @@
...
+			       	   <span class='tohide hide'>
+					       #{form @Comments.postComment(item.id, user.id)}
+						       	<textarea name="content" id="content"></textarea>
+						        <input type="submit" value="Comment" />
+						   #{/form}
+					   </span>
+				   </span>
+				 </span>
+	    	  </div>
+		  </span>
```
Comment (jtessler) [inline app/views/Application/news.html]: I think you may have replaced the space indentation with tabs here.

**Comment by jtessler** [comment]
Other than the indentation issue, this seems like a cool feature! Any way to write a Selenium test for it?

**Comment by amshali** [comment]
I like this. Can you replace more and less with those plus and minus figures?

**Comment by npetryk** [comment]
I could, but I think it's more natural in the post content to see a more and less. I've seen alot of website that opt for this particular text. 

Functionality wise though, the plus and minus figure will be automatically injected into a .triggershow and .triggerhide element if it is an img element

**Pull request merged**